### ISSUE TITLE
indent doesn't work with comment (jsdoc)

### ISSUE THREAD

**Issue by gitawego** [beautify-web/js-beautify#913]
for this code:

``` js
class test {
  method1() {
      let resp = null;
    }
    /**
     * @param {String} id
     */
  method2(id) {
    let resp2 = null;
  }
}
```

the code is not correctly formatted. I've to remove jsdoc to get it works as intended.

it happens when `indent-char = " "`

**Comment by bitwiseman** [comment]
How is it not correct?  Please provide this input, expected, and actual output.

**Comment by gitawego** [comment]
in my .jsbeautifyrc:
```
{
  "indent_size": 2,
  "indent_char": " ",
}
```
the correct code format should be like this:
![test_js_-_audit-server](https://cloud.githubusercontent.com/assets/438120/14420025/f0d67a9e-ffc9-11e5-9ea6-0468a1c7b860.jpg)

but with jsbeautify, it becomes this:
![test_js_-_audit-server](https://cloud.githubusercontent.com/assets/438120/14420043/08b43520-ffca-11e5-8f74-d57fae959efa.jpg)

the ``const`` of first method has 6 spaces instead of 4. the close brace of first method is aligned with the jsdoc of 2nd method, which is supposed to have 2 spaces instead of 4.

**Comment by IgorNovozhilov** [comment]
@gitawego, I have the same problem. beautification will work correctly if you leave a new line before the JSDoc description.
```js
class test {
   method1() {
      let resp = null;
   }

   /**
    * @param {String} id
    */
   method2(id) {
      let resp2 = null;
   }
}
```

@bitwiseman, I think that after the beautification, the code should look that way. Because method1 and method2 are functions. And in between functions adds a new line, as in the example below.

before beautification
```js
function f1() {}
function f2() {}
```

after beautification
```js
function f1() {}

function f2() {}
```

**Comment by bitwiseman** [comment]
This has the same cause as #1043 and would also benefit from the fix proposed in #1069.
